source/_components/climate.mqtt.markdown

```diff
@@ -54,6 +54,10 @@ initial:
   required: false
   type: number
   default: 21
+power_state_topic:
```

Comment:
Please place this below `power_command_topic` (line 93 of the original file so options are a bit grouped.
Feel free to add `power_state_template` there as well,
generel order of variables seem like command_topic, state_topic, state_template.